mutt/notify.c

```diff
@@ -37,11 +37,11 @@
 /// Lookup table for NotifyType
 /// Must be the same size and order as #NotifyType
```

Comment:
for the size part, you can use static_assert().

Reply:
good idea!